docs/user/requirements.rst

```diff
@@ -77,6 +71,21 @@ This can be done by running the following line of code in your preferred termina
 Keep in mind that this package is not required to run RocketPy, but it can be useful if you want to perform Environment Analysis.
 Furthermore, timezonefinder can only be used with Python 3.8+.
 
+Similar to timezonefinder, you can install other optional packages, including:
+
+- `windrose` : to allow for windrose plots when performing Environment Analysis,
+- `ipywidgets` : to allow for gifs generation when performing Environment Analysis,
+- `jsonpickle` : to allow for saving and loading of RocketPy objects.
+
+You can install these packages by running the following lines in your preferred terminal:
+
+.. code-block:: shell
+
+    pip install "windrose>=1.6.8"
+    pip install "ipywidgets>=7.6.3"
+    pip install jsonpickle
+
+
```

Comment:
And this should be included in the part above. It makes more sense to me

Reply:
Solved here: https://github.com/RocketPy-Team/RocketPy/pull/368/commits/8a45c7b896e5ef869ef2072b22e18e1e86f5e18a

Yes, indeed it improved clarity, thanks!